spatialmath/geo_geometry_test.go

```diff
@@ -50,26 +50,6 @@ func TestGeoGeometries(t *testing.T) {
 	test.That(t, testPoint, test.ShouldResemble, testGeoObst.Location())
 	test.That(t, testGeoms, test.ShouldResemble, testGeoObst.Geometries())
 
-	t.Run("Conversion from GeoGeometry to Protobuf", func(t *testing.T) {
```

Comment:
I don't hate moving this with the other proto; there is some sense to keeping all the proto conversions for geometries together.  However, having ownership of this structure spread across two packages is an antipattern we should avoid if possible so for that reason I think we should revert this code

Reply:
I had to move these otherwise we create a cyclical import between `spatialmath` and `referencframe` because these functions rely on `referenceframe.NewGeometryFromProto` but `referenceframe` imports the `spatialmath` library. I don't think there is a way to make this work without moving these.

<img width="1356" height="888" alt="Screenshot 2025-08-12 at 10 57 22 AM" src="https://github.com/user-attachments/assets/61895695-cc60-4923-a893-3af1886d2477" />

Reply:
Makes sense.  Resolving this comment